.eslintrc.cjs

```diff
@@ -1,26 +1,13 @@
 module.exports = {
+	parser: "@typescript-eslint/parser",
 	env: {
-		es2021: true,
 		node: true,
 	},
-	extends: ['xo', 'xo-typescript'],
-	parser: '@typescript-eslint/parser',
-	parserOptions: {
```

Comment:
Same with these

Reply:
Your local eslint setup might not be working, but these are all needed